### ISSUE TITLE
Support modern PBS resource syntax

### ISSUE THREAD

**Issue by bentsherman** [nextflow-io/nextflow#867]
I've been testing a nextflow workflow on a system which uses the PBS scheduler and I noticed that nextflow uses the old syntax for specifying resources on PBS. An example taken from the PBS executor test:
```
#PBS -l nodes=1:ppn=8
```

The "modern" syntax is as follows:
```
#PBS -l select=1:ncpus=8
```

Now I had thought that PBS Professional still supports the old syntax but according to my HPC admin, it doesn't, and indeed when I try to run processes that use multiple CPUs (such that this syntax is triggered), the processes fails because PBS doesn't accept the resource list. In particular I get this error from `qsub`:
```
qsub: Cannot be used with select or place: nodes
```

Even though nextflow didn't generate any other "select" or "place" arguments in the resource list. I can work around the problem using the `clusterOptions` directive but I think the modern syntax should still be supported. So could we change the PBS executor to use the modern "select/ncpus" syntax?

**Comment by pditommaso** [comment]
Unfortunately PBS world is extremely fragmented, there are many different distributions with subtle differences. Using the syntax you are suggesting would quite likely break the support for other users. 

Eventually it could be defined as config option. Is that the only setting preventing to use NF in your PBS cluster?

**Comment by bentsherman** [comment]
Yes I think all of the other fields work fine.

**Comment by pditommaso** [comment]
This is going to be addressed by #886

**Comment by pditommaso** [comment]
This has been included in version `18.10.1`. It requires to use of `pbspro` executor.
